docs/available_commands.md

```diff
@@ -80,6 +80,12 @@ The *target component* can be identified through any of the identifiable propert
 
 If *coordinates* are used to identify the target, they must match the component fully. In other words, if __only__ *name* is given, it will __only match__ components with that name which do __not__ contain *version* or *group* fields.
 
+In *coordinates* it is also possible to provide a range of versions using *version_range* parameter instead *version* following the [PURl specification](https://github.com/package-url/purl-spec/blob/version-range-spec/VERSION-RANGE-SPEC.rst) as referenced by [CycloneDX](https://cyclonedx.org/docs/1.6/json/#vulnerabilities_items_affects_items_versions_items_range).
+
+Further Information on the supported versioning schemas can be found here [univers documentation](https://pypi.org/project/univers/). Note that for every schema it is possible to provide a wildcard like *vers:generic/\** to allow all versions of the given schema.
+
+If *version* and *version_range* are provided, *version_range* will be ignored.
```

Comment:
This would clearly be an accidental error in the input data, I can't imagine any viable use case for having both fields. Wouldn't it be better to throw a useful error message here, informing the user that they have made a mistake instead of silently doing something which might not be what the user intended?

Reply:
That case is now raising an AppError

Reply:
The documentation should be updated accordingly.